Neos.Flow/Classes/I18n/Cldr/Reader/PluralsReader.php

```diff
@@ -150,6 +150,10 @@ public function getPluralForm($quantity, Locale $locale)
             return self::RULE_OTHER;
         }
 
+        if (!isset($this->rulesets[$locale->getLanguage()])) {
+            return self::RULE_OTHER;
+        }
+
         $ruleset = $this->rulesets[$locale->getLanguage()][$this->rulesetsIndices[$locale->getLanguage()]];
```

Comment:
What about easily:
`$ruleset = $this->rulesets[$locale->getLanguage()][$this->rulesetsIndices[$locale->getLanguage()]]; ?? null`

Reply:
Thanks, that works fine too. I have updated my commit.